src/orders/components/OrderUnfulfilledProductsCard/OrderUnfulfilledProductsCard.tsx

```diff
@@ -42,15 +46,40 @@ const OrderUnfulfilledProductsCard: React.FC<OrderUnfulfilledProductsCardProps>
           <TableHeader />
           <TableBody>
             {renderCollection(lines, line => (
-              <TableLine isOrderLine line={line} />
+              <TableLine isOrderLine line={line} isFulfilled={false} />
             ))}
           </TableBody>
         </ResponsiveTable>
         {canFulfill && (
           <CardActions>
-            <Button variant="text" color="primary" onClick={onFulfill}>
-              <FormattedMessage defaultMessage="Fulfill" description="button" />
-            </Button>
+            {noProductsAvailable ? (
+              <Tooltip
+                title={intl.formatMessage(messages.deletedVariantDetected)}
+                variant={"error"}
```

Comment:
Weird that linters wouldn't fix it
```suggestion
                variant="error"
```